win/packaging/extra.wxs.in

```diff
@@ -528,10 +528,10 @@
       AllowAdvertise='no'
       Level='1'
       Display='hidden'>
-      <ComponentRef Id='C.bin.mysql.exe'/>
-      <ComponentRef Id='C.bin.mysqladmin.exe'/>
-      <ComponentRef Id='C.bin.mysql_upgrade.exe'/>
-      <ComponentRef Id='C.bin.mysqlcheck.exe'/>
+      <ComponentRef Id='C.bin.mariadb.exe'/>
+      <ComponentRef Id='C.bin.mariadb_admin.exe'/>
+      <ComponentRef Id='C.bin.mariadb_upgrade.exe'/>
+      <ComponentRef Id='C.bin.mariadb_check.exe'/>
```

Comment:
Please, not.

Reply:
The tools, e.g  mariadb-upgrad.exe call another tools e.g mysql_check.exe call other tools, e.g mysql.exe . If you do not install mysql.exe, that mariadb-check will fail.  So, could yolu leave the old thing in place for now.

Reply:
Reverted in bd5310ad45dfbcec57f696b4ada04083786f94ec

Reply:
Reverted in bd5310ad45dfbcec57f696b4ada04083786f94ec